CRITICAL - 2024-01-13 14:27:41 --> ErrorException: require(../layouts/header.php): Failed to open stream: No such file or directory
in APPPATH/Views/nakes_site/tabel_data_pasien.php on line 2.
 1 APPPATH/Views/nakes_site/tabel_data_pasien.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require(../layouts/header.php): Failed to open stream: No such file or directory', '/Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Views/nakes_site/tabel_data_pasien.php', 2)
 2 APPPATH/Views/nakes_site/tabel_data_pasien.php(2): require()
 3 SYSTEMPATH/View/View.php(228): include('/Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Views/nakes_site/tabel_data_pasien.php')
 4 SYSTEMPATH/View/View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH/Common.php(1178): CodeIgniter\View\View->render('nakes_site/tabel_data_pasien', [], true)
 6 APPPATH/Controllers/Home.php(9): view('nakes_site/tabel_data_pasien')
 7 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\Home->index()
 8 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 14:28:13 --> ErrorException: require(../../layouts/header.php): Failed to open stream: No such file or directory
in APPPATH/Views/nakes_site/tabel_data_pasien.php on line 2.
 1 APPPATH/Views/nakes_site/tabel_data_pasien.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require(../../layouts/header.php): Failed to open stream: No such file or directory', '/Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Views/nakes_site/tabel_data_pasien.php', 2)
 2 APPPATH/Views/nakes_site/tabel_data_pasien.php(2): require()
 3 SYSTEMPATH/View/View.php(228): include('/Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Views/nakes_site/tabel_data_pasien.php')
 4 SYSTEMPATH/View/View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH/Common.php(1178): CodeIgniter\View\View->render('nakes_site/tabel_data_pasien', [], true)
 6 APPPATH/Controllers/Home.php(9): view('nakes_site/tabel_data_pasien')
 7 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\Home->index()
 8 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 14:28:15 --> ErrorException: require(../../layouts/header.php): Failed to open stream: No such file or directory
in APPPATH/Views/nakes_site/tabel_data_pasien.php on line 2.
 1 APPPATH/Views/nakes_site/tabel_data_pasien.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require(../../layouts/header.php): Failed to open stream: No such file or directory', '/Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Views/nakes_site/tabel_data_pasien.php', 2)
 2 APPPATH/Views/nakes_site/tabel_data_pasien.php(2): require()
 3 SYSTEMPATH/View/View.php(228): include('/Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Views/nakes_site/tabel_data_pasien.php')
 4 SYSTEMPATH/View/View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH/Common.php(1178): CodeIgniter\View\View->render('nakes_site/tabel_data_pasien', [], true)
 6 APPPATH/Controllers/Home.php(9): view('nakes_site/tabel_data_pasien')
 7 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\Home->index()
 8 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 14:42:35 --> Error: Class "App\Models\PasienModel" not found
in APPPATH/Controllers/PasienController.php on line 17.
 1 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\PasienController->tabel_data_pasien()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PasienController))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
ERROR - 2024-01-13 14:42:54 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:185
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(185): mysqli->real_connect('127.0.0.1', 'root', 'root', 'layanan_kesehat...', 3306, '', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 /Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Controllers/PasienController.php(18): CodeIgniter\BaseModel->findAll()
#7 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/CodeIgniter.php(941): App\Controllers\PasienController->tabel_data_pasien()
#8 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PasienController))
#9 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php(79): CodeIgniter\CodeIgniter->run()
#11 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('/Applications/X...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#4 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#5 /Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Controllers/PasienController.php(18): CodeIgniter\BaseModel->findAll()
#6 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/CodeIgniter.php(941): App\Controllers\PasienController->tabel_data_pasien()
#7 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PasienController))
#8 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php(79): CodeIgniter\CodeIgniter->run()
#10 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('/Applications/X...')
#11 {main}
CRITICAL - 2024-01-13 14:42:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pasien`', [], false)
 3 SYSTEMPATH/Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/PasienController.php(18): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\PasienController->tabel_data_pasien()
 7 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PasienController))
 8 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 14:52:57 --> ErrorException: Undefined array key "id_kategori"
in APPPATH/Views/tabel_data_pasien.php on line 51.
 1 APPPATH/Views/tabel_data_pasien.php(51): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_kategori"', '/Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Views/tabel_data_pasien.php', 51)
 2 SYSTEMPATH/View/View.php(228): include('/Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Views/tabel_data_pasien.php')
 3 SYSTEMPATH/View/View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1178): CodeIgniter\View\View->render('tabel_data_pasien.php', [], true)
 5 APPPATH/Controllers/PasienController.php(20): view('tabel_data_pasien.php', [...])
 6 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\PasienController->tabel_data_pasien()
 7 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PasienController))
 8 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 15:03:55 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "edit_data_pasien.php"
in SYSTEMPATH/Exceptions/FrameworkException.php on line 39.
 1 SYSTEMPATH/View/View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('edit_data_pasien.php')
 2 SYSTEMPATH/Common.php(1178): CodeIgniter\View\View->render('edit_data_pasien.php', [], true)
 3 APPPATH/Controllers/PasienController.php(53): view('edit_data_pasien.php', [...])
 4 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\PasienController->edit('1')
 5 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PasienController))
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 15:21:11 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "tambah_data_pasien.php"
in SYSTEMPATH/Exceptions/FrameworkException.php on line 39.
 1 SYSTEMPATH/View/View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('tambah_data_pasien.php')
 2 SYSTEMPATH/Common.php(1178): CodeIgniter\View\View->render('tambah_data_pasien.php', [], true)
 3 APPPATH/Controllers/PasienController.php(26): view('tambah_data_pasien.php')
 4 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\PasienController->tambah()
 5 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PasienController))
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 15:21:29 --> ErrorException: Undefined variable $pasien
in APPPATH/Views/tambah_data_pasien.php on line 14.
 1 APPPATH/Views/tambah_data_pasien.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $pasien', '/Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Views/tambah_data_pasien.php', 14)
 2 SYSTEMPATH/View/View.php(228): include('/Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Views/tambah_data_pasien.php')
 3 SYSTEMPATH/View/View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1178): CodeIgniter\View\View->render('tambah_data_pasien.php', [], true)
 5 APPPATH/Controllers/PasienController.php(26): view('tambah_data_pasien.php')
 6 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\PasienController->tambah()
 7 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PasienController))
 8 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 15:40:01 --> Error: Class "App\Models\RiwayatModel" not found
in APPPATH/Controllers/RiwayatController.php on line 11.
 1 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\RiwayatController->index()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RiwayatController))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 15:56:34 --> Error: Cannot use object of type stdClass as array
in APPPATH/Views/tabel_riwayat.php on line 46.
 1 SYSTEMPATH/View/View.php(228): include()
 2 SYSTEMPATH/View/View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH/Common.php(1178): CodeIgniter\View\View->render('tabel_riwayat', [], true)
 4 APPPATH/Controllers/RiwayatController.php(14): view('tabel_riwayat', [...])
 5 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\RiwayatController->index()
 6 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RiwayatController))
 7 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
ERROR - 2024-01-13 16:01:54 --> mysqli_sql_exception: Cannot delete or update a parent row: a foreign key constraint fails (`layanan_kesehatan`.`riwayat`, CONSTRAINT `riwayat_ibfk_1` FOREIGN KEY (`ID_Pasien`) REFERENCES `pasien` (`ID_Pasien`)) in /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(306): mysqli->query('DELETE FROM `pa...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `pa...')
#2 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `pa...')
#3 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(2782): CodeIgniter\Database\BaseConnection->query('DELETE FROM `pa...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Model.php(453): CodeIgniter\Database\BaseBuilder->delete()
#5 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/BaseModel.php(1117): CodeIgniter\Model->doDelete(Array, false)
#6 /Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Controllers/PasienController.php(79): CodeIgniter\BaseModel->delete(Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/CodeIgniter.php(941): App\Controllers\PasienController->delete('2')
#8 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PasienController))
#9 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php(79): CodeIgniter\CodeIgniter->run()
#11 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('/Applications/X...')
#12 {main}
CRITICAL - 2024-01-13 16:01:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot delete or update a parent row: a foreign key constraint fails (`layanan_kesehatan`.`riwayat`, CONSTRAINT `riwayat_ibfk_1` FOREIGN KEY (`ID_Pasien`) REFERENCES `pasien` (`ID_Pasien`))
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 SYSTEMPATH/Database/BaseBuilder.php(2782): CodeIgniter\Database\BaseConnection->query('DELETE FROM `pasien`
WHERE `ID_Pasien` IN :ID_Pasien:', [...], false)
 2 SYSTEMPATH/Model.php(453): CodeIgniter\Database\BaseBuilder->delete()
 3 SYSTEMPATH/BaseModel.php(1117): CodeIgniter\Model->doDelete([...], false)
 4 APPPATH/Controllers/PasienController.php(79): CodeIgniter\BaseModel->delete([...])
 5 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\PasienController->delete('2')
 6 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PasienController))
 7 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 16:01:54 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot delete or update a parent row: a foreign key constraint fails (`layanan_kesehatan`.`riwayat`, CONSTRAINT `riwayat_ibfk_1` FOREIGN KEY (`ID_Pasien`) REFERENCES `pasien` (`ID_Pasien`))
in SYSTEMPATH/Database/MySQLi/Connection.php on line 311.
 1 SYSTEMPATH/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `pasien`
WHERE `ID_Pasien` IN (\'2\')')
 2 SYSTEMPATH/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `pasien`
WHERE `ID_Pasien` IN (\'2\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2782): CodeIgniter\Database\BaseConnection->query('DELETE FROM `pasien`
WHERE `ID_Pasien` IN :ID_Pasien:', [...], false)
 4 SYSTEMPATH/Model.php(453): CodeIgniter\Database\BaseBuilder->delete()
 5 SYSTEMPATH/BaseModel.php(1117): CodeIgniter\Model->doDelete([...], false)
 6 APPPATH/Controllers/PasienController.php(79): CodeIgniter\BaseModel->delete([...])
 7 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\PasienController->delete('2')
 8 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PasienController))
 9 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 16:01:54 --> [Caused by] mysqli_sql_exception: Cannot delete or update a parent row: a foreign key constraint fails (`layanan_kesehatan`.`riwayat`, CONSTRAINT `riwayat_ibfk_1` FOREIGN KEY (`ID_Pasien`) REFERENCES `pasien` (`ID_Pasien`))
in SYSTEMPATH/Database/MySQLi/Connection.php on line 306.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(306): mysqli->query('DELETE FROM `pasien`
WHERE `ID_Pasien` IN (\'2\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `pasien`
WHERE `ID_Pasien` IN (\'2\')')
 3 SYSTEMPATH/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `pasien`
WHERE `ID_Pasien` IN (\'2\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2782): CodeIgniter\Database\BaseConnection->query('DELETE FROM `pasien`
WHERE `ID_Pasien` IN :ID_Pasien:', [...], false)
 5 SYSTEMPATH/Model.php(453): CodeIgniter\Database\BaseBuilder->delete()
 6 SYSTEMPATH/BaseModel.php(1117): CodeIgniter\Model->doDelete([...], false)
 7 APPPATH/Controllers/PasienController.php(79): CodeIgniter\BaseModel->delete([...])
 8 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\PasienController->delete('2')
 9 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PasienController))
10 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
ERROR - 2024-01-13 16:27:49 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`layanan_kesehatan`.`riwayat`, CONSTRAINT `riwayat_ibfk_1` FOREIGN KEY (`ID_Pasien`) REFERENCES `pasien` (`ID_Pasien`)) in /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(306): mysqli->query('INSERT INTO `ri...', 0)
#1 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ri...')
#2 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ri...')
#3 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ri...', Array, false)
#4 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 /Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Controllers/RiwayatController.php(33): CodeIgniter\Model->insert(Array)
#8 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/CodeIgniter.php(941): App\Controllers\RiwayatController->simpan()
#9 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RiwayatController))
#10 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php(79): CodeIgniter\CodeIgniter->run()
#12 /Applications/XAMPP/xamppfiles/htdocs/project_fix/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('/Applications/X...')
#13 {main}
CRITICAL - 2024-01-13 16:27:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`layanan_kesehatan`.`riwayat`, CONSTRAINT `riwayat_ibfk_1` FOREIGN KEY (`ID_Pasien`) REFERENCES `pasien` (`ID_Pasien`))
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 SYSTEMPATH/Database/BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `riwayat` (`riwayat_kesehatan`, `rekomendasi_gizi`, `created_at`) VALUES (:riwayat_kesehatan:, :rekomendasi_gizi:, :created_at:)', [...], false)
 2 SYSTEMPATH/Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Controllers/RiwayatController.php(33): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\RiwayatController->simpan()
 7 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RiwayatController))
 8 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 16:27:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`layanan_kesehatan`.`riwayat`, CONSTRAINT `riwayat_ibfk_1` FOREIGN KEY (`ID_Pasien`) REFERENCES `pasien` (`ID_Pasien`))
in SYSTEMPATH/Database/MySQLi/Connection.php on line 311.
 1 SYSTEMPATH/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `riwayat` (`riwayat_kesehatan`, `rekomendasi_gizi`, `created_at`) VALUES (\'adada\', \'dada\', \'2024-01-11\')')
 2 SYSTEMPATH/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `riwayat` (`riwayat_kesehatan`, `rekomendasi_gizi`, `created_at`) VALUES (\'adada\', \'dada\', \'2024-01-11\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `riwayat` (`riwayat_kesehatan`, `rekomendasi_gizi`, `created_at`) VALUES (:riwayat_kesehatan:, :rekomendasi_gizi:, :created_at:)', [...], false)
 4 SYSTEMPATH/Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Controllers/RiwayatController.php(33): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\RiwayatController->simpan()
 9 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RiwayatController))
10 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 16:27:49 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`layanan_kesehatan`.`riwayat`, CONSTRAINT `riwayat_ibfk_1` FOREIGN KEY (`ID_Pasien`) REFERENCES `pasien` (`ID_Pasien`))
in SYSTEMPATH/Database/MySQLi/Connection.php on line 306.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(306): mysqli->query('INSERT INTO `riwayat` (`riwayat_kesehatan`, `rekomendasi_gizi`, `created_at`) VALUES (\'adada\', \'dada\', \'2024-01-11\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `riwayat` (`riwayat_kesehatan`, `rekomendasi_gizi`, `created_at`) VALUES (\'adada\', \'dada\', \'2024-01-11\')')
 3 SYSTEMPATH/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `riwayat` (`riwayat_kesehatan`, `rekomendasi_gizi`, `created_at`) VALUES (\'adada\', \'dada\', \'2024-01-11\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `riwayat` (`riwayat_kesehatan`, `rekomendasi_gizi`, `created_at`) VALUES (:riwayat_kesehatan:, :rekomendasi_gizi:, :created_at:)', [...], false)
 5 SYSTEMPATH/Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Controllers/RiwayatController.php(33): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\RiwayatController->simpan()
10 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RiwayatController))
11 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 16:29:48 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "edit_data_riwayat.php"
in SYSTEMPATH/Exceptions/FrameworkException.php on line 39.
 1 SYSTEMPATH/View/View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('edit_data_riwayat.php')
 2 SYSTEMPATH/Common.php(1178): CodeIgniter\View\View->render('edit_data_riwayat.php', [], true)
 3 APPPATH/Controllers/RiwayatController.php(46): view('edit_data_riwayat.php', [...])
 4 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\RiwayatController->edit('7')
 5 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RiwayatController))
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 16:36:14 --> ErrorException: Undefined variable $pasien
in APPPATH/Views/edit_data_riwayat.php on line 14.
 1 APPPATH/Views/edit_data_riwayat.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $pasien', '/Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Views/edit_data_riwayat.php', 14)
 2 SYSTEMPATH/View/View.php(228): include('/Applications/XAMPP/xamppfiles/htdocs/project_fix/app/Views/edit_data_riwayat.php')
 3 SYSTEMPATH/View/View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1178): CodeIgniter\View\View->render('edit_data_riwayat.php', [], true)
 5 APPPATH/Controllers/RiwayatController.php(46): view('edit_data_riwayat.php', [...])
 6 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\RiwayatController->edit('7')
 7 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RiwayatController))
 8 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 21:05:36 --> Error: Class "App\Models\BeratBadanModel" not found
in APPPATH/Controllers/BeratBadanController.php on line 11.
 1 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\BeratBadanController->index()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\BeratBadanController))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
CRITICAL - 2024-01-13 21:06:00 --> BadMethodCallException: Call to undefined method App\Models\BeratBadanModel::getBeratBadantWithNamaPasien
in SYSTEMPATH/Model.php on line 874.
 1 APPPATH/Controllers/BeratBadanController.php(12): CodeIgniter\Model->__call('getBeratBadantWithNamaPasien', [])
 2 SYSTEMPATH/CodeIgniter.php(941): App\Controllers\BeratBadanController->index()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\BeratBadanController))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('/Applications/XAMPP/xamppfiles/htdocs/project_fix/public/index.php')
